Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

ann0see commented on _translator-files/po/de/Installation-for-Android.po:
```diff
@@ -1,20 +1,19 @@
-"PO-Revision-Date: 2022-08-16 16:56+0000\n"
-"Last-Translator: ann0see <please@seeGitHubOrWeblate.local>\n"
+"PO-Revision-Date: 2022-12-18 17:48+0000\n"
+"Last-Translator: ann0see <20726856+ann0see@users.noreply.github.com>\n"
```
The change from upstream works ;-). Probably the next release is the last release where we need to remove the e-mails manually.

ann0see commented on _translator-files/po/de/general.po:
```diff
@@ -43,7 +43,7 @@ msgstr "Eine Software von <a href=\"https://github.com/corrados\" target=\"_blan
 #, no-wrap
 msgid "Jamulus 3.9.1 addresses a MIDI controller-related bug that could result in the loss of old fader levels: If you used `--ctrlmidich`, you can start the app once with `--cleanuplegacyfadersettings` to try to convert the old format to the new one. This option will be removed in a future release. Read the <a href=\"https://github.com/jamulussoftware/jamulus/blob/master/ChangeLog\">Change Log</ ...
-msgstr ""
+msgstr "Jamulus 3.9.1 behebt einen Fehler im Zusammenhang mit MIDI-Controllern, der zum Verlust alter Faderpegel führen konnte: Wenn du `--ctrlmidich` verwendet hast, kannst du die App einmal mit `--cleanuplegacyfadersettings` starten, um zu versuchen, das alte Format in das neue zu konvertieren. Diese Option wird in einer zukünftigen Version entfernt. Weitere Informationen findest du im <a href= ...
```
Sounds bulky and not very native...

ann0see commented on _translator-files/po/fr/Include-Shared-Commands.po:
```diff
@@ -2,11 +2,12 @@
 # Copyright (C) YEAR Free Software Foundation, Inc.
 # This file is distributed under the same license as the PACKAGE package.
 # Olivier Humbert <please@seeGitHubOrWeblate.local>, 2022.
+# Olivier Humbert <trebmuh@tuxfamily.org>, 2022.
```
Duplicate needs to be removed

trebmuh replied:
It seems to be something which need to be removed in the source code here on github since I can't managed to do that through the weblate interface.

ann0see replied:
Yes. I think so too

ann0see commented on _translator-files/po/fr/Installation-for-Linux.po:
```diff
@@ -64,16 +65,15 @@ msgstr "{% include infobox_each_os.html %}"
+#, no-wrap
 msgid "**Upgrading?** You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first.\n"
-msgstr "Mise à jour ? Vous pouvez d'abord [sauvegarder votre configuration](Software-Manual#sauvegarde-de-jamulus)."
+msgstr "**Mise à jour ?** Vous pouvez d'abord [sauvegarder votre configuration](Software-Manual#sauvegarde-de-jamulus).\n"
```
Needing `\n` is not nice...

trebmuh replied:
The translation is following the original (English) message. the `\n` should be removed from the source code first.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Directories
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/
Translation: Jamulus/Software-Manual

ann0see committed:
Merge branch 'next-release' into weblate-jamulus-1-index

ann0see commented:
Looking at the CI log, the build didn't fully succeed after I resolved the conflicts. At least po4a gives error messages.

ann0see commented on _translator-files/po/de/general.po:
```diff
@@ -43,7 +43,8 @@ msgstr "Eine Software von <a href=\"https://github.com/corrados\" target=\"_blan
 #, no-wrap
 msgid "Jamulus 3.9.1 addresses a MIDI controller-related bug that could result in the loss of old fader levels: If you used `--ctrlmidich`, you can start the app once with `--cleanuplegacyfadersettings` to try to convert the old format to the new one. This option will be removed in a future release. Read the <a href=\"https://github.com/jamulussoftware/jamulus/blob/main/ChangeLog\">Change Log</a> ...
-msgstr ""
+msgstr "Jamulus 3.9.1 behebt einen Fehler im Zusammenhang mit MIDI-Controllern, der zum Verlust alter Faderpegel führen konnte: Wenn du `--ctrlmidich` verwendet hast, kannst du die App einmal mit `--cleanuplegacyfadersettings` starten, um zu versuchen, das alte Format in das neue zu konvertieren. Diese Option wird in einer zukünftigen Version entfernt. Weitere Informationen findest du im <a href= ...
```
```suggestion
msgstr "Jamulus 3.9.1 behebt einen Fehler im Zusammenhang mit MIDI-Controllern, der zum Verlust alter Faderpegel führen konnte: Wenn du `--ctrlmidich` verwendet hast, kannst du die App einmal mit `--cleanuplegacyfadersettings` starten, um zu versuchen, das alte Format in das neue zu konvertieren. Diese Option wird in einer zukünftigen Version entfernt. Weitere Informationen findest du im <a href=\"https://github.com/jamulussoftware/jamulus/blob/main/ChangeLog\">Changelog</a>."
```

ann0see commented on _translator-files/po/it/Command-Line-Options.po:
```diff
@@ -32,49 +33,48 @@ msgstr "wiki"
-msgstr ""
+msgstr "{% include_relative Include-Shared-Commands.md %}"
+
+#. type: Plain text
```
I hope this gets updated by Po4a automatically

ann0see commented on _translator-files/po/it/Contribution.po:
```diff
@@ -32,44 +33,42 @@ msgstr "wiki"
-msgid "See our [guidelines for getting involved in development](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md)."
-msgstr ""
+msgid "See our [guidelines for getting involved in development](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md)."
+msgstr "Vedi le nostre [linee guida per essere coinvolti nello sviluppo](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md)."
```
```suggestion
msgstr "Vedi le nostre [linee guida per essere coinvolti nello sviluppo](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md)."
```

ann0see commented on _translator-files/po/it/Contribution.po:
```diff
@@ -32,44 +33,42 @@ msgstr "wiki"
 #: ../wiki/en/Contribution.md:19
-msgid "See our [guidelines for getting involved in development](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md)."
-msgstr ""
+msgid "See our [guidelines for getting involved in development](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md)."
```
```suggestion
msgid "See our [guidelines for getting involved in development](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md)."
```

ann0see commented on _translator-files/po/it/Include-Shared-Commands.po:
```diff
@@ -2,53 +2,98 @@
+#. type: Plain text
+#: ../wiki/en/Include-Shared-Commands.md:9
+#, no-wrap
+msgid "`--jsonrpcport`            Enables JSON-RPC API server to control the app, set TCP port number (EXPERIMENTAL, APIs might change; only accessible from localhost). Please see [the JSON-RPC API Documentation file](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md).\n"
```
```suggestion
msgid "`--jsonrpcport`            Enables JSON-RPC API server to control the app, set TCP port number (EXPERIMENTAL, APIs might change; only accessible from localhost). Please see [the JSON-RPC API Documentation file](https://github.com/jamulussoftware/jamulus/blob/main/docs/JSON-RPC.md).\n"
```

ann0see commented on _translator-files/po/it/Include-Shared-Commands.po:
```diff
@@ -2,53 +2,98 @@
+#: ../wiki/en/Include-Shared-Commands.md:9
+#, no-wrap
+msgid "`--jsonrpcport`            Enables JSON-RPC API server to control the app, set TCP port number (EXPERIMENTAL, APIs might change; only accessible from localhost). Please see [the JSON-RPC API Documentation file](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md).\n"
+msgstr "`--jsonrpcport`            Consente al server API JSON-RPC di controllare l'app, impostare il numero di porta TCP (SPERIMENTALE, le API potrebbero cambiare; accessibile solo da localhost). Vedere il [file di documentazione dell'API JSON-RPC](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md).\n"
```
```suggestion
msgstr "`--jsonrpcport`            Consente al server API JSON-RPC di controllare l'app, impostare il numero di porta TCP (SPERIMENTALE, le API potrebbero cambiare; accessibile solo da localhost). Vedere il [file di documentazione dell'API JSON-RPC](https://github.com/jamulussoftware/jamulus/blob/main/docs/JSON-RPC.md).\n"
```

ann0see reviewed (commented):
Probably needs local testing

ann0see committed:
Remove occurances of master from translation
